### ISSUE TITLE
LAVA support for ConvT layers in LAVA-DL training

### ISSUE THREAD

**Issue by alexggener** [lava-nc/lava#619]

**Comment by bamsumit** [comment]
Hi @alexggener great. Early Q2 sounds fine. Drop me an email or message.

**Comment by bamsumit** [comment]
Here is an example conversion for basic convolution with stride 1:
```python
import numpy as np
import torch
import torch.nn.functional as F

for i in range(10):
    kernel_x = np.random.randint(7) + 1
    kernel_y = np.random.randint(7) + 1
    ch_in = np.random.randint(10) + 1
    ch_out = np.random.randint(10) + 1
    x_in = np.random.randint(100) + 1
    y_in = np.random.randint(100) + 1
    # stride = np.random.randint(5) + 1
    # padding = np.random.randint(5)
    # dilation = np.random.randint(5) + 1
    stride = 1
    padding = 0
    dilation = 1
    kernel = torch.rand([ch_in, ch_out, kernel_y, kernel_x])
    convT_input = torch.rand([1, ch_in, y_in, x_in])
    convT_output =  F.conv_transpose2d(convT_input, kernel, stride=stride, padding=padding, dilation=dilation)
    convT = F.conv2d(convT_input,
                     kernel.permute([1, 0, 2, 3]).flip([2, 3]),
                     padding=(kernel_y - 1, kernel_x - 1),
                     dilation=1)
    print(f'{i=} {torch.norm(convT - convT_output)=}')
```
Your task is to generalize it for arbitrary stride, padding and dilation and verify it.
